libethereum/Client.cpp

```diff
@@ -787,6 +788,16 @@ bool Client::submitSealed(bytes const& _header)
 void Client::rewind(unsigned _n)
 {
 	bc().rewind(_n);
+	onChainChanged(ImportRoute{});
```

Comment:
/libethereum/libethereum/Client.cpp:791: error: missing initializer for member 'dev::eth::ImportRoute::deadBlocks' [-Werror=missing-field-initializers]
  onChainChanged(ImportRoute{});
                             ^